Cargo.toml

```diff
@@ -88,8 +88,10 @@ tracing-subscriber = "0.2"
 url = "2.1"
 
 [dev-dependencies]
+bigdecimal = { version = ">= 0.0.10, < 0.2.0" }
 claim = "0.5"
 conduit-test = "0.9.0-alpha.4"
+diesel = { version = "1.4.0", features = ["numeric"] }
```

Comment:
the `numeric` feature is necessary because the `semver_triple` returned by `to_semver_no_prerelease()` is based on the `Numeric` SQL type.